### PR TITLE
Fix #2965 - Additional Properties set to False

### LINKED ISSUE #2965: Additional Properties should be False
**Is your feature request related to a problem? Please describe.**
Schemas

**Describe the solution you'd like**
Review all the JSON Schemas and add `additionalProperties: False` everywhere. We don't want schemas to become a Key Value store for anything,

**Describe alternatives you've considered**
No alternatives, we should fix this.

**Additional context**
Let's make sure there are no issues on the python side

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

This PR fixes https://github.com/open-metadata/OpenMetadata/issues/2965

We have done a review on all the JSON Schemas and added the missing `additionalProperties: false` to those entities and definitions that had it missing.

Java & python tests + sample ingestion run properly.

Thanks!

#
### Type of change :

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/operations/AirflowPipelineResourceTest.java [hunk 1]
```diff
@@ -547,7 +547,7 @@ private void validateGeneratedAirflowPipelineConfig(AirflowPipeline airflowPipel
     DatabaseService databaseService = helper(airflowPipeline).findEntity("service", DATABASE_SERVICE);
     DatabaseConnection databaseConnection = databaseService.getDatabaseConnection();
     DatabaseServiceMetadataPipeline metadataPipeline =
-        JsonUtils.convertValue(airflowPipeline.getPipelineConfig(), DatabaseServiceMetadataPipeline.class);
+        JsonUtils.convertValue(airflowPipeline.getPipelineConfig().getConfig(), DatabaseServiceMetadataPipeline.class);
```
Comment (pmbrull) [inline catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/operations/AirflowPipelineResourceTest.java:550]: Needed to add this when casting to `DatabaseServiceMetadataPipeline`, as the actual properties where nested in a `config` object that was now unnexpected

File: ingestion/examples/sample_data/topics/topics.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
 	"topics": [{
 			"name": "customer_events",
 			"description": "Kafka topic to capture the customer events such as location updates or profile updates",
-			"fullyQualifiedName": "sample_kafka.customer_events",
```
Comment (pmbrull) [inline ingestion/examples/sample_data/topics/topics.json:5]: Removed FQDN from topic sample data as that is not required for create APIs

**Review by mithmatt — APPROVED** [review]
I wonder if there is a way to configure all json schema objects under a path to have additionalProps set to false

**Comment by pmbrull** [comment]
> I wonder if there is a way to configure all json schema objects under a path to have additionalProps set to false

That would make things easier 😅

**Review by harshach — APPROVED** [review]

**Pull request merged**